config.yaml

```diff
@@ -58,17 +58,18 @@ contact: 'team@carpentries.org' # FIXME
 # - another-learner.md
 
 # Order of episodes in your lesson
-episodes: 
+episodes:
```

Comment:
I think your editor automatically removes trailing spaces. We can tolerate this for this PR, but it would be desirable to avoid such setting, in order to not to pollute revisions history with whitespace changes.